manim/utils/unit.py

```diff
@@ -0,0 +1,22 @@
+"""Implement the Unit class."""
+
+import numpy as np
+
+from .. import constants, config
+
+
+def m(val):
```

Comment:
What does this `m` stand for?

Reply:
Somewhere in the depths of the codebase, the current system of coordinates is called "MUnits".